src/main/webapp/app/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -55,6 +60,7 @@ export class ListOfComplaintsComponent implements OnInit {
         complaintResponse.subscribe(
             res => {
                 this.complaints = res.body;
+                this.complaintsToShow = this.complaints.filter(c => c.accepted === undefined);
```

Comment:
If the endpoint response takes a while, it might be that the user has changed the checkbox state for showAddressedComplaints.
Then the complaints are still filtered, even though showAddressedComplaints is true.
I think it would be best to disable the checkbox while loading.

Reply:
Good one, thanks!